### PR TITLE
Document environment variables.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: src/dev/env.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+There are a number of environment variables which control the behaviour of the
+yk system.
+
+## `YK_PRINT_IR`
+
+When `YK_PRINT_IR=1`, yk will print (to `stderr`) a textual version of each
+trace's LLVM IR prior to code generation.
+
+This variable is set by the `c_tests` suite for structural IR testing.
```
Comment (ltratt) [inline src/dev/env.md:11]: I don't think this sentence is useful: it suggests *only* the test suite should use this, but people might find other uses for it.

File: src/dev/env.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## `YK_PRINT_IR_SBS`
+
+When `YK_PRINT_IR_SBS=1`, yk will print (to `stderr`) a "side-by-side" listing
+of each trace's LLVM IR prior to code generation. The listing shows the
+instructions of the trace alongside the instructions from which they were
+derived (from the AOT IR stored in the interpreter binary). Inlining boundaries
+are also annotated on the listing.
+
+This is useful for debugging traces.
```
Comment (ltratt) [inline src/dev/env.md:21]: I'm not sure this sentence is helpful.

File: src/dev/env.md (hunk 1 above)
Comment (vext01) [reply src/dev/env.md:11]: That implication wasn't intentional. Shall we just remove it then, or?

Comment (ltratt) [reply src/dev/env.md:11]: Agreed.

Comment (ltratt) [reply src/dev/env.md:11]: Ping on this one.

**Comment by vext01** [comment]
Both comments now addressed. OK to squash?

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat

**Pull request merged**